# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436904789 14400
#      Tue Jul 14 16:13:09 2015 -0400
# Node ID 4847e3dd8e6abb85d1d1f7144bf1e3dc42372ef2
# Parent  1ea2d7d8abc76c3b0fcd3512d9eb2604da7955a0
fix file and directory names for doxyhtml targets

* doc/module.mk (doxyhtml): Correct name of Doxyfile.
(DIRSTAMP_FILES): Fix file name.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -31,23 +31,23 @@ if AMCOND_BUILD_DOCS
 EXTRA_DIST += \
   doc/texinfo.tex \
   doc/texmf.cnf
 
 EXTRA_DIST += \
   doc/doxyhtml/Doxyfile.in \
   doc/doxyhtml/README
 
-doxyhtml: doxyhtml/$(octave_dirstamp)
-	doxygen doxyhtml/Doxyfile
+doxyhtml: doc/doxyhtml/$(octave_dirstamp)
+	doxygen doc/doxyhtml/Doxyfile
 
 doxyhtml-maintainer-clean:
-	rm -rf doc/doxyhtml && rm -rf `ls | $(GREP) -v Doxyfile.in`
+	cd doc/doxyhtml && rm -rf `ls | $(GREP) -v Doxyfile.in`
 
-DIRSTAMP_FILES += doxyhtml/$(octave_dirstamp)
+DIRSTAMP_FILES += doc/doxyhtml/$(octave_dirstamp)
 
 GRAPH_PROP_TEXI_SRC = \
   doc/interpreter/plot-axesproperties.texi \
   doc/interpreter/plot-figureproperties.texi \
   doc/interpreter/plot-imageproperties.texi \
   doc/interpreter/plot-lineproperties.texi \
   doc/interpreter/plot-patchproperties.texi \
   doc/interpreter/plot-rootproperties.texi \
